spec/fluent.ebnf

```diff
@@ -56,32 +56,29 @@ block_placeable     ::= blank_block blank_inline? inline_placeable
  * syntax/abstract.mjs. */
 InlineExpression    ::= StringLiteral
                       | NumberLiteral
-                      | CallExpression
-                      | AttributeExpression
                       | VariantExpression
-                      | MessageReference
-                      | TermReference
-                      | VariableReference
+                      | ReferenceExpression
```

Comment:
Would it make sense to fold `VariantExpression` into `ReferenceExpression`, even if it's deprecated? It might be closer to how the tooling parser will handle it.

Reply:
I think it makes sense to have it explicit at this level.